.github/workflows/build.yml

```
@@ -295,6 +295,24 @@ jobs:
       working-directory: tests/FlatBuffers.Test.Swift
       run: sh SwiftTest.sh
 
+  build-swift-wasm:
+    name: Build Swift WASM
```

This capitalization is incorrect. The WebAssembly spec uses "Wasm" in its text and all related materials.
```suggestion
    name: Build Swift Wasm
```